pkg/apis/core/validation/validation_test.go

```diff
@@ -21304,9 +21223,7 @@ func TestValidateWindowsHostProcessPod(t *testing.T) {
 
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.WindowsHostProcessContainers, testCase.featureEnabled)()
-
-			opts := PodValidationOptions{AllowWindowsHostProcessField: testCase.featureEnabled}
```

Comment:
if we're dropping testing of this validation option (because the associated feature is always on now), drop the `AllowWindowsHostProcessField` field from the `PodValidationOptions` type and all code that checks `Enabled(features.WindowsHostProcessContainers)` as well

Reply:
Sure - I'll make these updates today.

Reply:
@liggitt I made the requested updates - PTAL and thanks!